src/create_learner.Rmd

```diff
@@ -464,6 +464,34 @@ getFeatureImportanceLearner.classif.randomForestSRC = function(.learner, .model,
 ```
 
 
+## Integrating out-of-bag predictions in the [%mlr] framework
+
+Some learners like random forest use bagging. Bagging means that the learner consists of a ensemble of several base learners and each base learner is trained with a different subsample or bootstrap sample from all observations. For predicting an observation, 
+that was used in the training process, one can predict the observation with all base learners, where this observation was not used 
```

Comment:
The sentence starting with "For predicting an observation, ..." seems a little complicated.
My proposal would be: "A prediction made for an observation in the original data set using only base learners not trained on this particular observation is called out-of-bag (OOB) prediction."